packages/material-ui/src/TablePagination/TablePagination.d.ts

```diff
@@ -24,8 +24,18 @@ export interface TablePaginationTypeMap<P, D extends React.ElementType> {
       labelRowsPerPage?: React.ReactNode;
       nextIconButtonProps?: Partial<IconButtonProps>;
       nextIconButtonText?: string;
-      onChangePage: (event: React.MouseEvent<HTMLButtonElement> | null, page: number) => void;
+      /**
+       * Deprecated. Will be removed in v5. Please use the onPageChange prop instead.
+       * @deprecated
+       */
+      onChangePage?: (event: React.MouseEvent<HTMLButtonElement> | null, page: number) => void;
```

Comment:
I've changed the old prop to not be required and made the new prop required. I guess this is the correct change.

Reply:
Although this can be considered breaking change 😕